.github/workflows/go.yml

```diff
@@ -40,16 +42,39 @@ jobs:
         run: ./test_all.sh
 
   quick-tests:
+    strategy:
+      matrix:
+        containerGoVer:
+          - "1.20"
+          - "1.21-rc"
     runs-on: ubuntu-latest
-    container: golang:1.20
+    container: golang:${{ matrix.containerGoVer }}
     steps:
     - uses: actions/checkout@v3
       with:
         persist-credentials: false
 
-
     - name: Go Build
       run: go build -v ./...
 
     - name: Unit Tests
       run: go test -v ./...
+
+  gvc:
+    strategy:
+      matrix:
+        containerGoVer:
+          - "1.20"
+          - "1.21-rc"
+    runs-on: ubuntu-latest
+    container: golang:${{ matrix.containerGoVer }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Install govulncheck
+        run: go install golang.org/x/vuln/cmd/govulncheck@latest
```

Comment:
I mentioned over in the Boulder PR that you can combine this into a single `go run` command.